src/definitions_cxx.hpp

```diff
@@ -152,6 +152,7 @@ enum class MidiLearn : uint8_t {
 	SECTION,
 	INSTRUMENT_INPUT,
 	DRUM_INPUT,
+	LOAD_BUTTON,
 };
```

Comment:
Do we really want to add this learn + load button shortcut to learn the global midi command? We don't do this with any of the other global midi commands.

Reply:
I thought it was handy as we have other learnable commands on buttons like Record on the record button. But if you feel it is not needed i can remove it